### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by theleanbow** [blueagler/QuillBot-Premium-Crack#42]
It gives the error:
Using premium features require token,  please chose a token
The token from telegram doesn't work either

**Comment by Tisanthan** [comment]
not working

**Comment by theleanbow** [comment]
@Tisanthan yes, i have done the sign in part, yet i am still getting this error while using modes other than standard and fluency (they are available on free version as well)
![image](https://user-images.githubusercontent.com/66841515/222755920-f6e90678-f7ab-4306-b3f6-8233a68ab212.png)
Haven't tested with word limit, but i think that one is working at least.
Checked on the recent version 2.0.1 as well, same issue

**Comment by aegon2** [comment]
I'm having the same issue 
![Screenshot (29)](https://user-images.githubusercontent.com/126893932/222880213-117fc21c-ebfa-4315-abcf-f56712bbf92d.png)

**Comment by Elvis143BRA** [comment]
Same problem here. Any fixes?

**Comment by Emma571-tech** [comment]
I installed latest premium crack 2.0.1 version as well, still telling me to upgrade to premium, please i need a solution to it thanks you.

**Comment by VekTor64** [comment]
I also installed the latest premium crack 3.0.1 version, it is not working still and asking me to upgrade to premium. Please what should I do

**Comment by CreativeThrill** [comment]
i mean it works now as of the  latest 3.0.2 update  although there is this  Use an credential in the Credentials manager to use single-paraphrase Error #50 red box popup but heres a script created by @ASharockT that removes it 

A simple tampermonkey script:
// ==UserScript==
// @name QuillBot-Premium-Crack Alert Remove
// @namespace http://tampermonkey.net/
// @Version 2024-05-30
// @description try to take over the world!
// https://github.com/author AST
// @match https://quillbot.com/*
// @ICON https://www.google.com/s2/favicons?sz=64&domain=quillbot.com
// @grant none
// ==/UserScript==

(function() {
'use strict';
// Your code here...
setInterval(function(){setTimeout(AlertRemove,1000);})
function AlertRemove(){
var Alert;
Alert=document.querySelector('.notistack-SnackbarContainer');
Alert.remove();
}
})();
